config/settings.yml

```diff
@@ -88,9 +88,8 @@ active_features: []
 
 reject_refuse_messaging_released_at: <%= DateTime.new(2018, 4, 24, 23, 05, 0).change(offset: '+100') %>
 
-# agfs_fee_reform_release_date: <%= Date.new(2018, 4, 1) %>
-agfs_fee_reform_release_date: <%= ENV["ENV"].in?(['api-sandbox', 'demo', 'development']) ? Date.new(2018, 1, 1) : Date.new(2018, 04, 1) %>
-agfs_scheme_11_release_date: <%= ENV["ENV"].in?(['gamma', 'staging', 'disaster']) ? Date.new(2018, 12, 31) : Date.new(2018, 12, 1) %>
+agfs_fee_reform_release_date: <%= Date.new(2018, 4, 1) %>
+agfs_scheme_11_release_date: <%= Date.new(2018, 12, 31) %>
```

Comment:
Generally :+1: for this, but should we let the software vendors know?

Reply:
will do. 👍

Reply:
vendors warned of this in #software-vendor-adp